docs/use.md

```diff
@@ -102,3 +170,61 @@ To do so, follow these steps:
 - Assign the token to an environment variable called `GITHUB_ACCESS_TOKEN`.
   If you run `github-activity` and this variable is defined, it will be used.
   You may also pass a token via the `--auth` parameter (though this is not the best security practice).
+
+## Use the Python API
+
+You can do most of the above from Python as well.
+This is not as well-documented as the CLI, but should have most functionality available.
+
+### Generate markdown changelogs with the Python API
+
+For generating markdown changelogs from Python, here's an example:
+
+```
+from github_activity import generate_activity_md
+
+markdown = generate_activity_md(
+    target="executablebooks/github-activity",
+    since="2023-01-01",
+    until="2023-12-31",
+    auth="your-github-token",
+    kind=None,
```

Comment:
```suggestion
    kind=None,
    auth="your-github-token",
    tags=None,
```
Might as well include all arguments in the same order as the code
https://github.com/executablebooks/github-activity/blob/c6e278f53ddf1faafa8ddfa460698325ecbc5840/github_activity/github_activity.py#L340-L352